File: javascript/1016.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var tempo = parseInt(lines.shift());
var vCarroX = 60;
var vCarroY = 90;
var calc = 60 * tempo / (vCarroY - vCarroX);
console.log(`${calc} minutos`);

File: Java14/1016.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);
        int tempo = Integer.parseInt(in.readLine());
        int calc= 60 * tempo / 30;
        System.out.printf("%d minutos\n",calc);
	}
}